bin/server.js

```diff
@@ -1,5 +1,7 @@
 var express = require('express');
+var cors = require('cors');
 var app = express();
+app.use(cors());
```

Comment:
how does additional lines improve readability here?  what is advantage of this change?